### ISSUE TITLE
Just a kind thank you. How come?

### ISSUE THREAD

**Issue by D1no** [f111fei/react-native-unity-view#8]
I did not have the time playing around with it yet, but marrying unity and react native was a long wish on which I spend a lot of desk research. Its not easy. Therefore, its amazing that you made it work.

I would really like to know: Whats this / your background? Are you guys using it in production?

Thanks a lot

**Comment by f111fei** [comment]
Yes, We are using the module in our AR project. Unity+Vuforia show the 3D model and AR object，RN show UI and basic logic. Maybe it will be a few months before you can see the app.
